packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useCallback, SetStateAction, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Portal from '@leafygreen-ui/portal';
+import { Global } from '@emotion/core';
```

Comment:
Can't use this, must use injectGlobal like @DesignerDave's using in his Syntax component.

Reply:
👍